lib/msf/core/exploit/remote/kerberos/ticket/storage/write_mixin.rb

```diff
@@ -36,7 +36,7 @@ def store_ccache(ccache, options = {})
       message << "MIT Credential Cache ticket saved to #{path}"
       print_status(message)
 
-      nil
+      { path: path }
```

Comment:
I think we should return the persisted object here, but `store_loot` doesn't do that either - so this is at least a compromise/point of extension if we want to add additional keys in the future